### PR TITLE
[E2E] Consolidate `question` group (assorted tweaks)

### PR DESCRIPTION
**Pull request by nemanjaglumac**
### Status
PENDING REVIEW

### What does this PR accomplish?
- Assorted tweaks for tests in the `question` E2E group
- Prepares the ground for a more meaningful refactoring of the slowest E2E group

### Note
Single commits are logical units and are much easier to follow and to review than the final diff.

### CODE REVIEW

File: frontend/test/metabase/scenarios/question/new.cy.spec.js [hunk 1]
```diff
@@ -162,55 +162,55 @@ describe("scenarios > question > new", () => {
     });
   });
 
-  describe("ask a (simple) question", () => {
```
Comment (gusaiani) [inline frontend/test/metabase/scenarios/question/new.cy.spec.js:172]: Nice.

**Review by npfitz — APPROVED** [review]
I checked through the commits individually and everything seemed okay to me.

File: frontend/test/metabase/scenarios/question/new.cy.spec.js (hunk 1 above)
Comment (nemanjaglumac) [reply frontend/test/metabase/scenarios/question/new.cy.spec.js:172]: That was a relic from the past. There are no simple and custom questions anymore.
Although, we still didn't deal with https://github.com/metabase/metabase/issues/19677

File: frontend/test/metabase/scenarios/question/nested.cy.spec.js [hunk 2]
```diff
@@ -532,6 +534,27 @@ describe("scenarios > question > nested", () => {
       );
     });
   });
+
+  it("should be able to use integer filter on a nested query based on a saved native question (metabase#15808)", () => {
```
Comment (gusaiani) [inline frontend/test/metabase/scenarios/question/nested.cy.spec.js:538]: In some places we have the `reproductions` dir inside a `scenarios` dir.

Here, the repro is directly in a file inside `scenarios` but not in a `reproductions` dir.

How do you decide which one goes where?

File: frontend/test/metabase/scenarios/question/new.cy.spec.js (hunk 1 above)
Comment (gusaiani) [reply frontend/test/metabase/scenarios/question/new.cy.spec.js:172]: Exactly, it's all in there and imo we should clean it up.

**Pull request merged**